src/Microsoft.Health.Fhir.Core/Features/Operations/Reindex/ReindexJobTask.cs

```diff
@@ -126,96 +124,207 @@ public async Task ExecuteAsync(ReindexJobRecord reindexJobRecord, WeakETag weakE
                     _reindexJobRecord.Count = countOnlyResults.TotalCount.Value;
```

Comment:
We don't seem to be using this Count value anywhere. We can avoid executing a query to get the total count.

Reply:
It is true, we don't use it yet.  I missed implementing an update to the progress property which uses the count to track overall total progress. For larger jobs I think it will be very useful, so I would like to keep it for now.  I will add a bug to make sure we implement the progress.